flow-sample/src/main/java/com/example/flow/view/TabletMasterDetailRoot.java

```diff
@@ -66,6 +66,7 @@ public TabletMasterDetailRoot(Context context, AttributeSet attrs) {
         if (countDown == 0) {
           disabled = false;
           wrapped.onTraversalCompleted();
+          ((IsMasterView) masterContainer.getCurrentChild()).setSingleChoiceMode();
```

Comment:
Hello

I just signed the agreement.

In this pr, my thought is that in the single view mode (not master-detail view mode), the single choice mode seems weird. It works pretty well in master-detailed mode since it implies which view is being selected.

What I want to do is to disable the single choice mode in normal view mode and enable that in master detail view mode.

Hmmm I got some idea all of a sudden. May make some tiny modifications.

Anyway I am really expecting the new FLOW lib ^^